src/components/DappBrowser/Homepage.tsx

```diff
@@ -37,13 +37,14 @@ import { useBrowserStore } from '@/state/browser/browserStore';
 import { DndProvider, Draggable, DraggableGrid, DraggableGridProps, UniqueIdentifier } from '../drag-and-drop';
 import { EasingGradient } from '../easing-gradient/EasingGradient';
 import { useBrowserContext } from './BrowserContext';
-import { getNameFromFormattedUrl } from './utils';
+import { getNameFromFormattedUrl, normalizeUrlWorklet } from './utils';
```

Comment:
Dont see `normalizeUrlWorklet` being used here

Reply:
ah yep was able to remove that chunk of code that used it, will clean up good catch